### DISCUSSION TITLE
Announcing distributed functions for Azure Static Web Apps

### DISCUSSION THREAD

**Discussion by thomasgauvin** [Azure/static-web-apps#1429] [Announcements]
Hi everyone!

We’re excited to announce Distributed Functions, a new feature for Static Web Apps that dynamically distributes your managed functions closer to your users to improve your app performance and decrease latency. 

When a region gets a sufficiently high traffic load and your Static Web Apps resource has distributed functions enabled, Static Web Apps will distribute your managed functions to that region and direct traffic accordingly. This can help reduce your network latency by up to 70%, which can be quite noticeable when your web application has global audience and relies on quick response times to handle personalization, routing or authorization. 

Distributed functions applies to your production environment, and is available for Standard SKU Static Web Apps. Currently, distributed functions is not available for Next.js sites but we are working on it!

During the preview of distributed functions, we’ll continue to tweak distributed functions’ trigger conditions to ensure it’s effectiveness. Currently, the request load on your backend functions is polled regularly to determine whether the region has sufficient load to trigger distribution. 

We hope you'll give distributed functions a try and let us know what you think. We're always listening to your feedback and suggestions on how to make Static Web Apps better! 🚀

Check out the announcement blog post for more details: https://techcommunity.microsoft.com/t5/apps-on-azure-blog/announcing-distributed-functions-for-azure-static-web-apps/ba-p/4083151

**Comment by danbasszeti** [comment]
This has been one of the feature's I was most hoping for since I experimented with this using an enormously complicated ARM template! Excellent stuff, looking forward to using it!
